cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -245,6 +245,16 @@ using optional_dremel_view = thrust::optional<detail::dremel_device_view const>;
  * second letter in both words is the first non-equal letter, and `a < b`, thus
  * `aac < abb`.
  *
+ * @note: The operator overloads in sub-class `element_comparator` are templated via the
+ *        `type_dispatcher` to help select an overload instance for each column in a table.
+ *        So, `cudf::is_nested<Element>` will return `true` if the table has nested-type columns,
+ *        but it will be a runtime error if template parameter `has_nested_columns != true`.
```

Comment:
Sorry, I just realized that this tag (here and in all other places) should not have `:`.
```suggestion
 * @note The operator overloads in sub-class `element_comparator` are templated via the
 *       `type_dispatcher` to help select an overload instance for each column in a table.
 *       So, `cudf::is_nested<Element>` will return `true` if the table has nested-type columns,
 ...
```